db/migrate/20230605110212_readd_training_record_states_as_table.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class ReaddTrainingRecordStatesAsTable < ActiveRecord::Migration[6.1]
```

Comment:
Spelling mistake... ReaddTrainingRecordStatesAsTable > ReadTrainingRecordStatesAsTable

Reply:
That was my attempt at spelling re-add 😅